plugins/woocommerce-admin/client/products/sections/images-section.scss

```diff
@@ -0,0 +1,79 @@
+.woocommerce-product-form {
+	&__images {
+		margin-bottom: 0;
+		box-shadow: unset;
+		.woocommerce-media-uploader {
+			text-align: center;
+			&__label {
+				display: flex;
+				flex-direction: column;
+				align-items: center;
+			}
+			img {
+				margin-bottom: $gap-small;
+				width: $gap-larger + $gap-larger;
+			}
+		}
+		.components-card__body {
+			padding: 0;
+		}
+	}
+	&__images.has-images {
+		.woocommerce-media-uploader {
```

Comment:
I would leave this code here, since these styles will be applied depending on whether the Card that contains the component `MediaUploader` and `ImageGallery` has a class or not.

Reply:
Good idea, if it's ok with you I'll tackle this in a follow-up PR.